h2o-algos/src/main/java/hex/schemas/GLMModelV3.java

```diff
@@ -31,9 +31,16 @@
     @API(help="Standardized Coefficient Magnitudes")
     TwoDimTableV3 standardized_coefficient_magnitudes;
 
-    @API(help="Lambda minimizing the objective value, only applicable with lambd search")
+    @API(help="Lambda minimizing the objective value, only applicable with lambda search or when arrays of alpha and " +
+            "lambdas are provided")
     double lambda_best;
 
+    @API(help="Alpha minimizing the objective value, only applicable when arrays of alphas are given ")
+    double alpha_best;
+
+    @API(help="submodel index minimizing the objective value, only applicable for arrays of alphas/lambda ")
+    int best_submodel_index; // denote the submodel index that yields the best result
```

Comment:
this is exposed only for tests right? Otherwise I don't see how this could be any use for the user.

Reply:
This index is useful when you do a glm.getGLMRegularizationPath(mL), this index will be the one that gives the best performance.